rpcs3/Emu/RSX/Program/CgBinaryFragmentProgram.cpp

```diff
@@ -189,8 +188,8 @@ template<typename T> std::string CgBinaryDisasm::GetSrcDisAsm(T src)
 			}
 			break;
 		}
+		break;
```

Comment:
Nested switches is nasty work. This switch is so unnecessary too, its just an if-else.
Actually, just ignore this file, it should be deleted at some point and the logic merged in with the real disassembler.